docs/CONFIG.md

```diff
@@ -4543,6 +4543,11 @@ TxConfirmTimeout = '30s' # Default
 SkipPreflight = true # Default
 Commitment = 'confirmed' # Default
 MaxRetries = 0 # Default
+FeeEstimatorMode = 'fixed' # Default
+MaxComputeUnitPrice = 1000 # Default
+MinComputeUnitPrice = 0 # Default
+DefaultComputeUnitPrice = 0 # Default
```

Comment:
WDYT about suffixing with the modifiers?
```suggestion
ComputeUnitPriceMax = 1000 # Default
ComputeUnitPriceMin = 0 # Default
ComputeUnitPriceDefault = 0 # Default
```

Reply:
I like that! i will update the configs

On a separate note: the EVM configs don't follow that haha `EvmGasPriceDefault` + `EvmMaxGasPriceWei` + `EvmMinGasPriceWei` (maybe something to fix in the future 🤷🏼 )